cloudflare/resource_cloudflare_ruleset_test.go

```diff
@@ -1329,6 +1379,68 @@ func testAccCheckCloudflareRulesetRateLimit(rnd, name, zoneID, zoneName string)
       expression = "(http.request.uri.path matches \"^/api/\")"
       description = "example http rate limit"
       enabled = true
-		}
-	}`, rnd, name, zoneID, zoneName)
+    }
+  }`, rnd, name, zoneID, zoneName)
+}
+
+func testAccCheckCloudflareRulesetActionParametersMultipleSkips(rnd, name, zoneID, zoneName string) string {
+	return fmt.Sprintf(`
+  resource "cloudflare_ruleset" "%[1]s" {
+    zone_id  = "%[3]s"
+    name        = "%[2]s"
+    description = "%[1]s ruleset description"
+    kind        = "zone"
+    phase       = "http_request_firewall_managed"
+
+    rules {
+      action = "skip"
+      action_parameters {
+        rulesets = ["efb7b8c949ac4650a09736fc376e9aee"]
+      }
+      expression = "(cf.zone.name eq \"domain.xyz\" and http.request.uri.query contains \"skip=rulesets\")"
+      description = "skip Cloudflare Manage ruleset"
+      enabled = true
+    }
+
+    rules {
+      action = "skip"
+      action_parameters {
+        # efb7b8c949ac4650a09736fc376e9aee is the ruleset ID of the Cloudflare Managed rules
+        rules = {
+          "efb7b8c949ac4650a09736fc376e9aee" = "5de7edfa648c4d6891dc3e7f84534ffa,e3a567afc347477d9702d9047e97d760"
```

Comment:
i don't love that `TypeMap` only allows simple types as values however this is the next best thing to having it as a list of strings and just perform the translation in the provider.